modules/misc/news.nix

```
@@ -2029,6 +2029,25 @@ in
           A new module is available: 'programs.foot'.
         '';
       }
+
+      {
+        time = "2021-05-19T01:26:16+00:00";
+        message = ''
+          The xmonad module now compiles the configuration beforing
+          linking the binary to the place xmonad expects to find
+          the compiled configuration (the binary).
+
+          This breaks recompilation of xmonad (i.e. the 'q' binding or
+          'xmonad --recompile').
+
+          If this behavior is undesirable, instead of using the option
+          'xsession.windowManager.xmonad.config', you may set the path
+          of the configuration file directory with
+          'home.file.".xmonad/config.hs".text = "content of the file"'
+          or 'home.file.".xmonad/config.hs".source = ./path-to-config'.
```

Maybe it's just late, but I think that this sentence is a bit hard to parse as-is. Maybe something along these lines would be more clear?

```suggestion
          If this behavior is undesirable, do not use the
          'xsession.windowManager.xmonad.config' option. Instead, set the
          contents of the configuration file with
          'home.file.".xmonad/config.hs".text = "content of the file"'
          or 'home.file.".xmonad/config.hs".source = ./path-to-config'.
```